terraform/api_gateway.tf.json.template.py

```diff
@@ -345,24 +345,22 @@ def for_domain(cls, domain):
                         }
                     }
                 }
+            },
+            'aws_lambda_function_event_invoke_config': {
+                function_name: {
+                    'function_name': '${aws_lambda_function.%s.function_name}' % function_name,
+                    'maximum_retry_attempts': 0
+                }
+                for function_name in [
+                    *(
+                        (f'indexer_{lm}' for lm in ('forward_alb_logs', 'forward_s3_logs'))
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14collection-literals

and see my diff for a trick to make this less noisy.